src/model/ModelBase.js

```diff
@@ -303,4 +302,4 @@ class ModelBase {
     }
 }
 
-export default new ModelBase();
+export default ModelBase;
```

Comment:
This seems more correct, and I think this might be why `Model` set the `ModelBase` object exported here on its own prototype, so it should be safe ... I think.